CHANGELOG.md

```diff
@@ -14,6 +14,9 @@
 
 ### Fixed
 
+- Fixed a bug that prevented the `--show-log-origin` option from printing the
```

Comment:
sorry to be pedantic but would this sound better?
"Fixed a bug that ignored the `--show-log-origin` option, preventing to print the source code file of the log messages"